dash/backend/src/modules/cluster/controllers/cluster.controller.ts

```diff
@@ -209,6 +209,40 @@ export class ClusterController {
         return cluster;
     }
 
+    @Put(':clusterId/activateWebhook')
+    @AllowedAuthorityLevels(Authority.SUPER_ADMIN, Authority.ADMIN)
+    @UseGuards(AuthGuard, AuthorityGuard)
+    @ApiResponse({
+        status: 200,
+        schema: UPDATE_CLUSTER_SCHEMA_RESPONSE
+    })
+    @UseInterceptors(AuditLogInterceptor)
+    async activateWebhook(@Body() activateWebhook: { value: boolean }, @Param('clusterId') clusterId: number): Promise<ClusterDto> {
+        const previousCluster = await this.clusterService.getClusterById(clusterId);
+        const currentCluster = await this.clusterService.activateWebhook(previousCluster, activateWebhook.value);
+        delete previousCluster.kubeConfig;
+        delete currentCluster.kubeConfig;
+        currentCluster.metadata = await this.clusterService.calculateClusterMetaData(previousCluster, currentCluster);
+        return currentCluster;
+    }
+
+    @Put(':clusterId/activateImageScanningEnforcement')
```

Comment:
Let's make the endpoints kebab case to be consistent with the others in the project.
activateWebhook > activate-webhook

activateImageScanningEnforcement > activate-image-scanning-enforcement

Reply:
changed the endpoints case.